CHANGELOG.md

```diff
@@ -23,6 +23,8 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 
 - [#1284](https://github.com/improbable-eng/thanos/pull/1284) Add support for multiple label-sets in Info gRPC service. This deprecates the single `Labels` slice of the `InfoResponse`, in a future release backward compatible handling for the single set of Labels will be removed. Upgrading to v0.6.0 or higher is advised.
 
+- [#1314](https://github.com/improbable-eng/thanos/pull/1314) Updates `prometheus/client_golang`. Remove usages of [deprecated methods](https://github.com/prometheus/client_golang/blob/505eaef017263e299324067d40ca2c48f6a2cf50/prometheus/http.go#L122) by introducing a wrapper package which mimics the same behaviour while using new APIs and fixes stated issues. As a result, removes `http_request_duration_microseconds` (Summary) and adds `http_request_duration_seconds` (Histogram).
```

Comment:
I would focus on user facing issues, so literaly metrics changes. From user standpoint in does not matter what client library we use (: Can we trim this?

Reply:
Of course, I'll trim it.